### PR TITLE
Sync image definitions with lxc-ci

### PR DESCRIPTION
**Pull request by MusicDin**
Cherry picked from [lxc/lxc-ci](https://github.com/lxc/lxc-ci)

### CODE REVIEW

**Comment by tomponline** [comment]
@MusicDin were these tested as i note: https://github.com/lxc/lxc-ci/issues/813

**Comment by simondeziel** [comment]
Considering the source and destination repos are both under Apache-2.0 license, I don't think we should carry the `License: Apache-2.0` on those cherry-picked commits. @tomponline what's your opinion on this?

**Comment by tomponline** [comment]
> Considering the source and destination repos are both under Apache-2.0 license, I don't think we should carry the `License: Apache-2.0` on those cherry-picked commits. @tomponline what's your opinion on this?

I've got an outstanding question for the LXD management team as to whether this should be licensed as AGPL 3.0 or Apache 2.0. 

However either way I don't see a problem with carrying the license line for imported cherry-picks.
I would prefer not to have to start having more complex cherry-picking guidelines where the process is different for different repos.

**Comment by tomponline** [comment]
> @MusicDin were these tested as i note: [lxc/lxc-ci#813](https://github.com/lxc/lxc-ci/issues/813)

@MusicDin ?

**Comment by MusicDin** [comment]
Yep, I'm still testing the recipes. Will update the PR with required cherry picks once done. Moving to draft.

File: images/ubuntu.yaml [hunk 1]
```diff
@@ -346,6 +346,7 @@ files:
   - kinetic
   - lunar
```
Comment (simondeziel) [inline images/ubuntu.yaml:347]: Time to drop kinetic (22.10) and lunar (23.04) as they are both EOL https://fridge.ubuntu.com/2023/12/15/ubuntu-23-04-lunar-lobster-reaches-end-of-life-on-january-25-2024/

Maybe there's a commit doing just that that needs to be cherry-picked?

**Review by simondeziel — APPROVED** [review]

**Comment by simondeziel** [comment]
I'm not sure [images/ubuntu: Re-enable legacy cloud-init in Ubuntu](https://github.com/canonical/lxd-ci/pull/69/commits/839baf6f0024c89ab536711ec5689866d9848a85) is actually needed as I'd assume this was done because cloud-init expects a LXD data source instead of an Incus one?

Anyway, if we do need it we should mangle the changelog with `s/Incus/LXD/g`.

**Comment by MusicDin** [comment]
There is no particular reason for those being split. I'll merge.

**Comment by MusicDin** [comment]
Merged to #71

**Pull request closed without merging**